pkg/commands/standard/raw_test.go

```diff
@@ -0,0 +1,47 @@
+package standard
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"testing"
+)
+
+func TestGetJsonBodyFromFlags(t *testing.T) {
+	for i, testcase := range []struct {
+		fileName         string
+		bodyString       string
+		params           map[string]string
+		expectedBody     string
+		expectedMimeType string
+	}{
+		{"raw_test_body.json", "", nil, "{\"test\": \"yes it is\"}\n", "application/json"},
```

Comment:
tip: You can use raw strings ` so you don't have to escape "

Reply:
I know but  didn't get that to work with whitespacing or something so I gave up on that...